typesafe_conductr_cli/conduct_load.py

```diff
@@ -24,8 +24,8 @@ def load(args):
         bundleId = response_json['bundleId']
 
         print("Bundle loaded.")
-        print("Start bundle with: cli/conduct run {}".format(bundleId))
-        print("Unload bundle with: cli/conduct unload {}".format(bundleId))
-        print("Print ConductR info with: cli/conduct info")
+        print("Start bundle with: conduct run{} {}".format(args.cli_parameters, bundleId))
```

Comment:
We can't add a space here, because it would be a double space when `args.cli_parameters` is empty, which is the case when no `host` nor `port` is specified.